availda/verify/verifier.go

```diff
@@ -0,0 +1,76 @@
+package verify
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/rollkit/go-da"
+	"github.com/stackrlabs/go-daash/availda"
+	"github.com/stackrlabs/go-daash/availda/verify/bindings/vectorverifier"
+)
+
+type DAVerifier struct {
+	daClient         *availda.DAClient
+	ethClient        *ethclient.Client
+	bridgeContract   common.Address
+	verifierContract common.Address
+}
+
+func NewDAVerifier(configPath string, ethEndpoint string, bridgeContract string, verifierContract string) (*DAVerifier, error) {
+	client, err := availda.New(configPath)
```

Comment:
can we not reuse the same DA client we initialize in the main and pass it as a parameter

Reply:
Hmm yes we can do that.